planner/core/logical_plan_builder.go

```diff
@@ -608,7 +619,10 @@ func (b *PlanBuilder) buildProjectionField(id, position int, field *ast.SelectFi
 		colName = field.AsName
 	} else {
 		// Other: field is an expression.
-		colName = b.buildProjectionFieldNameFromExpressions(field)
+		var err error
+		if colName, err = b.buildProjectionFieldNameFromExpressions(field); err != nil {
+			return nil, errors.Trace(err)
```

Comment:
Once the error is with stack, we don't need `errors.Trace` either.

Reply:
ok